guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,29 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Linux Unified Key Setup (LUKS) during host provisioning.
+Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign the partition table to your host group or select it manually during provisioning.
+
+Encrypted disks on hosts protect data at rest.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
```

Comment:
Is "select" the right word to choose here? I've seen "select" used with checkboxes on this project. In this case, wouldn't "click" be a better word?

Reply:
@asteflova 
This is a dual list selector. See Max's [comment](https://github.com/theforeman/foreman-documentation/pull/2871#discussion_r1533389323) with screenshot. 

Here is an example of usage: "From the **Available options** list, select the required options and then click the arrow to move the options to the **Chosen options** list." 

So "select" is the right verb because you select the option and then click the arrow to "associate" it with the OS. I'm not thrilled with the word "associate", but Max says it's understood in the context.